spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaBinderConfiguration.java

```diff
@@ -73,20 +64,17 @@
  * @author Mark Fisher
  * @author Ilayaperumal Gopinathan
  * @author Henryk Konsek
+ * @author Gary Russell
  */
 @Configuration
 @ConditionalOnMissingBean(Binder.class)
-@Import({ KryoCodecAutoConfiguration.class, PropertyPlaceholderAutoConfiguration.class,
-		KafkaBinderConfiguration.KafkaPropertiesConfiguration.class })
+@Import({ PropertyPlaceholderAutoConfiguration.class})
```

Comment:
Can someone explain why do we need this at all if it is presented via Spring Boot by default?

Reply:
Issue created: https://github.com/spring-cloud/spring-cloud-stream-binder-kafka/issues/211